spec/graphql/types/albums/album_search_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+require 'support/graphql/fixtures/graphql_album_helper'
```

Comment:
This should be already required by the line 26 of the rails_helper.rb file.

Reply:
fixed